tests/tests_all_options.yml

```
@@ -4,6 +4,7 @@
   gather_facts: true
   vars:
     ssh_c: {}
+    pkg_mgr: "{{ 'dnf' if ansible_facts['distribution_version'] | int > 7 else 'yum' }}"
```

```suggestion
    pkg_mgr: "{{ (ansible_facts['distribution_version'] | int > 7) | ternary('dnf', 'yum' }}"
```
Also - if `__ssh_is_ostree` is true, then you cannot run any commands which uninstall or reinstall packages

---

Fixed the long lines and added condition for the ostree. Hope it will be enough